client/lib/route/test/index.js

```diff
@@ -1,17 +1,17 @@
-/* eslint-disable vars-on-top */
-require( 'lib/react-test-env-setup' )();
-
 /**
  * External dependencies
  */
-var expect = require( 'chai' ).expect;
+import { expect } from 'chai';
 
 /**
  * Internal dependencies
  */
-var route = require( '../index' );
+import useFakeDom from 'test/helpers/use-fake-dom';
+import route from '../';
```

Comment:
Do we need `useFakeDom` here at all?  I looked through `client/lib/route/` and didn't see anything that would load React, which makes sense because this code is mainly just manipulating URL strings.